scipy/signal/_savitzky_golay.py

```diff
@@ -62,8 +62,8 @@ def savgol_coeffs(window_length, polyorder, deriv=0, delta=1.0, pos=None,
     >>> savgol_coeffs(5, 2)
     array([-0.08571429,  0.34285714,  0.48571429,  0.34285714, -0.08571429])
     >>> savgol_coeffs(5, 2, deriv=1)
-    array([  2.00000000e-01,   1.00000000e-01,   2.00607895e-16,
-            -1.00000000e-01,  -2.00000000e-01])
+    array([ 2.00000000e-01,  1.00000000e-01,  2.07548111e-16, -1.00000000e-01,
+           -2.00000000e-01])
```

Comment:
Scanning over `refguide_check.py`, my initial guess is that this isn't opted-in for testing, especially given that this is a "private module." That could be a matter for another PR though. If it is worth updating the docstring for new results, it is perhaps worth having it tested.

Reply:
I haven't been following the developments in the doc-building tools very closely.  The names `savgol_coeffs` and `savgol_filter` end up in the `signal` namespace.  Are they not checked like everything else?

Reply:
Ah, maybe they are then--I can always check locally by breaking the docstring intentionally in case these deviations weren't sufficient, and then running refguide.

Reply:
> I can always check locally by breaking the docstring intentionally in case these deviations weren't sufficient, and then running refguide.

OK, I just tried that.  In the `savgol_coeffs` docstring, I locally changed the last output value from 6.0 to 16.0.  When I ran `python refguide_check.py --doctests --verbose signal`, the example failed:

```
Running checks for 1 modules:
scipy.signal ..........................................................................................................................................................................................................................................................................................................F...................................

============
scipy.signal
============

Non-deprecated objects in __all__: 176
Objects in refguide: 156

scipy.signal.savgol_coeffs
--------------------------

File "/Users/warren/miniconda36/lib/python3.6/site-packages/scipy-1.3.0.dev0_-py3.6-macosx-10.7-x86_64.egg/scipy/signal/_savitzky_golay.py", line 82, in savgol_coeffs
Failed example:
    c.dot(x)
Expected:
    16.0
Got:
    6.0


ERROR: refguide or doctests have errors
```

When the docstring has the correct value, the script runs with no errors.